src/igvfd/schemas/construct_library_set.json

```diff
@@ -2,7 +2,7 @@
     "title": "Construct Library Set",
     "$id": "/profiles/construct_library_set.json",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "description": "A file set containing raw data files resulting from sequencing of the library delivered to the sample. For example, a guide RNA library.",
+    "description": "Construct Library Sets are file-sets that capture raw data files (e.g. FASTQs) that result from sequencing of a library prior to the delivery of this library into the samples being investigated. For example sequencing results of guide RNAs after cloning them but prior to delivery. The results represent the sequencing output of the guides that are then introduced into cells, but may not always correspond to what exact guides ended up being deliered or expressed. Not all Construct Library Sets will end up having FASTQs or any other files in them. Example: if the lab chooses not to sequence their guide library prior to delivery, no FASTQs will be required in that case. Construct Library Sets should not be associated with any samples because it is designed to capture the library prior to the delivery and hence, has no relation to the Sample that will get the plasmids eventually.",
```

Comment:
* Same comment about schema capitalization, and no hyphen in "file set"
* Typo "deliered" instead of "delivered"
* I suggest not using colon separated things like "Example:" if we can write, "For example, if the lab chooses..."